Xamarin.Forms.Xaml.UnitTests/DesignTimeLoaderTests.cs

```diff
@@ -451,4 +525,27 @@ public void CssStyleTargetingMissingTypeNotAppliedToFallbackType()
 			Assert.That(myButton.BackgroundColor, Is.Not.EqualTo(Color.Blue));
 		}
 	}
+
+	public class InstantiateThrows
+	{
+		public InstantiateThrows()
+		{
+			throw new InvalidOperationException();
+		}
+
+		public InstantiateThrows([Parameter("Value")] string value)
```

Comment:
this is internal API, no one is supposed to use it

Reply:
Oh, hah, I thought I remembered seeing it documented somewhere previously. Guess we can delete this one.